Yosemite/Yosemite/Model/Orders/StoredOrderSettings.swift

```diff
@@ -27,6 +30,9 @@ public struct StoredOrderSettings: Codable, Equatable {
             if let dateRangeFilter = dateRangeFilter, dateRangeFilter.filter != .any {
                 total += 1
             }
+            if let _ = productFilter {
```

Comment:
Nit:
```suggestion
            if productFilter != nil {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/12089/commits/b677b9837fa63f29e59f1b19cbd275e6d7860fbb